apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -11,6 +11,12 @@ const EVENTS = {
   ABANDON_SECTION_SETUP_SIGN_IN_EVENT: 'Abandon Section Setup Sign In',
   TEACHER_LOGIN_EVENT: 'Teacher Login',
 
+  // School Association
+  // Update School Info Dialog
+  UPDATE_SCHOOL_INFO_DIALOG_SHOWN: 'Update School Info Dialog Shown',
+  CONFIRM_SCHOOL_CLICKED: 'Confirm School Clicked',
+  UPDATE_SCHOOL_CLICKED: 'Update School Clicked',
+
```

Comment:
I know this isn't in the product spec but curious if it's work having an `UPDATE_SCHOOL_INFO_DIALOG_CLOSED` event? Or do we think we can figure that out by calculating the number of users that didn't confirm or update?

Reply:
That sounds like a good idea!

Reply:
Added!